Makefile

```diff
@@ -27,7 +27,7 @@ help:
 .PHONY: services
 services: args?=up -d
 services: python
-	@tox -qe docker-compose -- $(args)
```

Comment:
I've renamed the `docker-compose` tox env to `dockercompose` because tox actually sees `docker-compose` as two separate "factors" (`docker` and `compose`) which isn't what we want. Did the same for `pip-compile` too